code/game/objects/items/holy_weapons.dm

```diff
@@ -273,7 +273,7 @@
 	
 
 /obj/item/nullrod/attack_self(mob/user)
-	if(user.mind && (user.mind.holy_role) && !reskinned)
+	if(user.mind && (user.mind.holy_role) && !reskinned && type == /obj/item/nullrod)
```

Comment:
already spawned null rods should have reskinned set then

Reply:
You want me to go through every single null rod subtype and mark reskinned on them?

Reply:
And checking for admin spawn won't work either, as admins commonly spawn in base nullrods for various reasons that chaplains are expected to be able to reskin

Reply:
> You want me to go through every single null rod subtype and mark reskinned on them?

yeah

Reply:
peak shitcode